sw/device/lib/base/BUILD

```diff
@@ -30,7 +30,7 @@ cc_library(
 )
 
 cc_library(
-    name = "math_builtins",
+    name = "math_polyfills",
```

Comment:
Nit: change the first commit message to refer to `math_polyfills` instead of `math_builtins_no_builtins`.